libkbfs/test_stallers.go

```diff
@@ -38,6 +39,166 @@ const (
 	StallableMDPutUnmerged           StallableMDOp = "PutUnmerged"
 )
 
+type naïveStallContext struct {
```

Comment:
Calling this `Context` (and using `ctx` for it in the code) is a bit confusing, since that's usually reserved for `context.Context`.  I know we do that in the DSL tests, but maybe we can avoid that here by just calling this `naïveStallOp`?